_pages/en/it-picture-medium-term.md

```diff
@@ -26,124 +26,146 @@ The foundational pieces for "Government as a service" vision are available and i
-The build statuses of projects are visible to the department.
+Chaos Engineering is being considered for use to build confidence in systems' capabilities to withstand unexpected conditions.
+
+Source code, assets, configurations, or other documents part of a project's development and deployment are stored in a version control system that records changes to files.
+Changes trigger automated tests and a build process that creates packages (executable, container, virtual machine, ...) that can be deployed to any environment (testing or production) in an automated way.
+Operations and infrastructure have moved towards automation as much as possible, being able to shift workloads and runtime from one service provider to another as it needs to.
+Builds are numbered, repeatable and being run on demand by teams giving them quick feedback on their small batches of work.
+Team members prioritize review of coworkers' changes to ensure that they don't have to wait days to get merged and deployed.
+The build status of projects is visible to the department.
 Teams have access to adequate test data to run automated tests for projects.
 
 Enterprise Architecture (EA) enables agility by empowering teams to choose tools and technologies, rapidly responding to new standards adoption, as well as providing light reference architectures.
-EA enables ESDC to re-use investments via capability-base planning and influences where the department's investments needs to be allocated.
-Technology stacks are periodically reviewed as part of team retrospectives (Agile), EA and security to critically evaluate how well the current tools address requirements.
+EA enables ESDC to re-use investments via capability-based planning and influences where the department's investments need to be allocated.
+Technology stacks are periodically reviewed as part of team retrospectives (Agile).
+EA and security periodically evaluate how well current tools address requirements.
 New technologies are also proactively investigated and opportunities are provided to discuss and demonstrate new technologies for the standard technology stacks.
+The Total Cost of Ownership (TCO), including exit costs, is calculated as part of solution options analysis.
+
+The security team is involved in the design phase of all projects and a security review is now a gating factor for releasing the design to the development stage.
+Security tests are built into the automated testing process and identify common security vulnerabilities.
+Also, the Privacy by design approach is taken into account throughout the whole engineering process.
 
-The security team is involved in the design phase for all projects and a security review is now a gating factor for releasing the design to the development stage.
-Security tests are built into the automated testing process and can identify common security vulnerabilities.
+Teams must experience the pain of own processes and procedures.
```

Comment:
@jaysonmc Not sure "pain" is the most appropriate term. What if the process/procedure is not painful? ;)
Could be simply _Teams must experience their..._

Reply:
I thought about that when I put it in ... we could put experience in, but I was thinking that it's just obfuscating the point. The use of them experiencing it is that if it sucks then they will notice it and be incentivized to fix it. 

It's the same idea behind making devs responsible for ops. They'll write better software if they have to wake up at 3am on a Saturday because they wrote bad code during the week. They're feeling the pain of their bad decisions. 

Guess you could rephrase it to say "experience" their own bad decisions, but what we really mean is the pain or self inflicted inconvenience of their bad decisions

I shall let the group decide, of course. But that's my take :)

Reply:
Je dirais :
Teams must experience their own processes and procedures to properly appreciate their shortcomings.

Reply:
Thanks for the suggestion! I took it and added "potential shortcomings".